src/SamplesApp/UITests.Shared/UITests.Shared.projitems

```diff
@@ -4044,6 +4052,15 @@
     <Compile Update="$(MSBuildThisFileDirectory)Windows_UI_Xaml_Controls\ToggleSwitchControl\ToggleSwitch_IsOn.xaml.cs">
       <DependentUpon>ToggleSwitch_IsOn.xaml</DependentUpon>
     </Compile>
+    <Compile Update="C:\Dev\Uno\src\SamplesApp\UITests.Shared\Windows_Devices\AccelerometerTests.xaml.cs">
```

Comment:
`$(MSBuildThisFileDirectory)`

Reply:
Thank you for noticing this, I have no clue how this got there